### ISSUE TITLE
5.1 not working on fresh install

### ISSUE THREAD

**Issue by RelicCornhusk** [mariotaku/moonlight-tv#454]
### Moonlight Version

1.6.18

### Device Type

webOS TV

### Device Model

LG C3

### Device OS

webOS 8.4.0-16

### Host OS

Windows

### Host Software

Sunshine

### Host Setup

Intel Core i5-12600K, AMD Radeon 6700XT, 16g RAM

### Bug Description

I have tried many things already but I cannot output 5.1 audio from my Denon receiver when using Moonlight. It works otherwise for any other app on LG TV. I already set my TV audio to passthrough. Also I have set it on the Moonlight app to 5.1.

### Reproduction Steps

To test, I've been opening steam big picture and opening any game. I've also tried modifying the steam streaming speakers audio configuration from stereo to 5.1 within windows while streaming.

### Expected Behavior

Streaming 5.1 audio.

**Comment by mariotaku** [comment]
Did you enable "5.1 surround" in "Audio Settings" in Moonlight?

**Comment by RelicCornhusk** [comment]
> Did you enable "5.1 surround" in "Audio Settings" in Moonlight?

Yes, even tested some other audio backends, but ended up leaving at auto since not all have the 5.1 option.

**Comment by mariotaku** [comment]
What about surround sound with this website? https://www2.iis.fraunhofer.de/AAC/multichannel.html

**Comment by mariotaku** [comment]
What about streaming desktop? I'll try with my gear too.

**Comment by RelicCornhusk** [comment]
Same result as before when testing with Desktop instead of Steam

**Comment by RelicCornhusk** [comment]
I think I got it to work. Here is what I did. I went to a different app on webOS and played 5.1 content. It seems like the Denon receiver (S760H) switched to 5.1 at that moment and then stayed at 5.1. so I went to moonlight and sure enough, the content played at 5.1. I expected it to show PCM on the receiver's screen, but it shows Dolby Digital Surround (DSur). Is that how it should be?

Maybe in the future if this happens again, I'll try switching back and forth from stereo to auto on my receiver to see if it will match the content being streamed.
